modules/javafx.graphics/src/main/java/com/sun/prism/mtl/MTLFBOTextureData.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.sun.prism.mtl;
+
+class MTLFBOTextureData extends MTLTextureData {
+    MTLFBOTextureData(MTLContext context, long texPtr, long size) {
+        super(context, texPtr, size);
+    }
+
+    @Override
+    public void dispose() {
```

Comment:
There is something wrong here: this method does not call `super.dispose()`.  Is it by design (a comment would be nice then).

The `MTLTextureData.dispose()` calls `MTLResourceFactory.releaseTexture(pTexture);` which will not be called id the context is disposed of.